### DISCUSSION TITLE
Jaeger helm chart

### DISCUSSION THREAD

**Discussion by yaztriumph** [jaegertracing/jaeger#3920] [Q&A]
When installing helm chart, if I enabled allinone do I disable other sections?

**Comment by birdx0810** [comment]
Yes! As mentioned in the AllInOne installation section [here](https://artifacthub.io/packages/helm/jaegertracing/jaeger#installing-the-chart-using-the-all-in-one-executable-and-in-memory-storage).

Cheers.
